README.md

```diff
@@ -78,52 +78,18 @@ You can also use `Ctrl` + `Alt` + `Shift` + `M` shortcut to open it.
 
-This branch layout can be automatically discovered based on the state of your git repository by the `Discover Branch Layout` action.
-It constructs a layout from around 10 most recently used branches.
-This action is proposed in Git Machete tab in case of an empty or nonexistent `machete` file,
-but you can also run it any time from IntelliJ's `Search Everywhere` (double Shift) by typing `Discover Branch Layout`.
-
-![](docs/discover.gif)
-
-
-### Edit machete file
-
-The `machete` file can be also edited manually. Let's look at the structure of this file based on the example below:
-```
-develop
-    allow-ownership-link PR #123
-        build-chain
-    call-ws PR #124
-master
-    hotfix/add-trigger
-```
-`develop`, `allow-ownership-link`, `build-chain`, `call-ws`, `master` and `hotfix/add-trigger` are branch names.
-Two of them, `allow-ownership-link` and `call-ws`, have a custom annotation &mdash;
-it's an arbitrary description displayed next to the given branch (in this case, pull request numbers).
-
-The relation between these branches is determined by indentations &mdash; here single indent is 4 spaces, but a tab can be used as well.
-
-In the example above, branches `allow-ownership-link` and `call-ws` are children of `develop`, while `build-chain` is a child of `allow-ownership-link`.
-`master`, in turn, is the parent of `hotfix/add-trigger`.
-`develop` and `master` are root branches.
-
-When the branch layout is created, Git Machete indicates a relation between the parent branch and each of its child branches.
+For each branch, Git Machete indicates a relation between this branch and each of its child branches.
 If the edge between them is green that means the child branch is in sync with its parent branch &mdash; in other words, there are no commits in the parent branch that don't belong to the child.
 But if there are some commits in the parent branch that are **not** reachable from the child, then the edge is red &mdash; you need to [rebase](#rebase) child branch onto the parent.
+Edge can also be gray and yellow &mdash; that means respectively the branch was merged to the parent, or a fork point can't be determined automatically.
 
-Machete file editor will help you with managing the `machete` file: it underlines any errors (bad indentation or nonexistent branches) and proposes branch names based on local repository branches.
-When file editing is done, you can click the button in the top right corner of the file editor to refresh the machete branch layout.
-
-![](docs/machete_file_editor.gif)
+![](docs/sample_graph.png)
 
-For more information about the `machete` file, look at the [reference blog post](https://medium.com/virtuslab/make-your-way-through-the-git-rebase-jungle-with-git-machete-e2ed4dbacd02).
+As we can see in example above `hotfix/add-trigger` is in sync with `master`.
+`call-ws` is **not** in sync with `develop` and `drop-constraint` is **not** in sync with `call-ws`.
+`build-chain` was merged into `develop`.
 
```

Comment:
I'd still add some link to the Edit section so that it's clear that the branch layout can be arbitrarily modified by a user (i.e. is not fixed in stone)